progress-bar.dust

```diff
@@ -0,0 +1,19 @@
+<div {?id}id="{.}"{/id} class="c-progress-bar c--{state} {?isRadial}c--radial{/isRadial} {class}" role="progressbar" aria-valuenow="{value}" aria-valuemin="0" aria-valuemax="1" aria-valuetext="{label}" data-adaptivejs-component="stencil-progress-bar">
```

Comment:
I’d like to start standardizing the order of common attributes, with `class` followed by `id`, then others. This is minor, but it will help to keep things easy to scan.